components/hero/Hero.tsx

```diff
@@ -0,0 +1,110 @@
+                    </svg>
+                    <span className="relative">
+                        {" "}
+                        Study{" "}
+                        <span className="text-black dark:text-white">& </span>
+                        contribute{" "}
+                    </span>
+                </span>{" "}
+                to Bitcoin open source development
+            </h1>
+            <p className="mx-auto mt-6 max-w-2xl text-lg tracking-tight text-black-700 dark:text-black-300">
+                The Bitcoin Dev Project is the go-to resource for developers to
+                get inspired by and learn bitcoin and related technologies to
+                kickstart and grow their career in Free and Open Source
+                Software.
+            </p>
+            <div className="mt-10 flex justify-center gap-x-6">
+                <Link href="/bitcoin-core">
+                    <button
+                        type="button"
+                        className="py-1 pl-4 duration-200 pr-2 border-orange-500 bg-orange-500 hover:text-black h-12 hover:bg-white text-white justify-between uppercase border-2 hover:border-white rounded-full inline-flex items-center gap-12"
+                    >
+                        Start Learning
+                        <div className="bg-black h-8 w-8 text-white inline-flex items-center justify-center rounded-full">
+                            {" "}
+                            <span>
+                                {" "}
+                                <svg
+                                    xmlns="http://www.w3.org/2000/svg"
+                                    className="w-5 h-5"
+                                    viewBox="0 0 32 32"
+                                >
+                                    <path
+                                        fill="currentColor"
+                                        d="m18 6l-1.43 1.393L24.15 15H4v2h20.15l-7.58 7.573L18 26l10-10L18 6z"
+                                    ></path>
+                                </svg>
+                            </span>{" "}
+                        </div>
+                    </button>
+                </Link>
+            </div>
+            <div className="mt-36 lg:mt-44">
+                <p className="font-display text-base text-black-800 dark:text-black-100">
+                    We're building products for Bitcoiners
+                </p>
+                <ul
+                    role="list"
+                    className="mt-8 flex items-center justify-center gap-x-8 sm:flex-col sm:gap-x-0 sm:gap-y-10 xl:flex-row xl:gap-x-12 xl:gap-y-0"
+                >
+                    {[
+                        [
```

Comment:
Given that there's an object (navigation in `footer.tsx`) you created to house something like this https://github.com/bitcoin-dev-project/bitcoin-dev-project/pull/83/files#diff-bd3f659ee78d8c923c9975bdb8842b2a346ff04023e42fe215c0d080ef5cf115R2, we could merge both arrays into one json object in a separate file and differentiate them with their keys so that they can be imported anywhere in the code. e.g:
```js
const globalObjData = {
     "socials" : [ ..... ],
     "products": [ ..... ]
}
```

Reply:
@jrakibi I know it's quite a lot of changes requested but I think you skipped this